charming/src/series/sankey.rs

```diff
@@ -99,6 +99,9 @@ pub struct Sankey {
     #[serde(skip_serializing_if = "Option::is_none")]
     emphasis: Option<Emphasis>,
 
+    #[serde(skip_serializing_if = "Option::is_none")]
+    layoutIterations: Option<u64>,
```

Comment:
[snake case](https://github.com/yuankunzhang/charming/pull/29/files#r1399539778) :smiley:

Reply:
True, that is how i started but then the translation into the javascript object uses the wrong name as it seems to require an identical name.

Reply:
But there is
 https://github.com/yuankunzhang/charming/blob/caad3c0917992be5316a1a8ffa4621d7bcea385b/charming/src/series/sankey.rs#L64

so it should be renamed :thinking:

Reply:
I've adapted the case to snake_case!